Modules/TOF/src/TaskRaw.cxx

```diff
@@ -47,8 +47,23 @@ void RawDataDecoder::rdhHandler(const o2::header::RAWDataHeader* rdh)
 {
   mCounterRDH[rdh->feeId & 0xFF].Count(0);
 
+  //Case for the RDH word "fatal"
   if ((rdh->detectorField & 0x00010000) != 0) {
     mCounterRDH[rdh->feeId & 0xFF].Count(1);
+    LOG(WARNING) << "RDH flag \"fatal\" error occurred in crate " << static_cast<int>(rdh->feeId & 0xFF);
```

Comment:
InfoLogger is used in some places and FairLogger in some others. It is ok for me, just raising your awareness about it.

Reply:
Thanks! We will address it in the next PR